tests/testthat/test_validate_fn.R

```diff
@@ -858,6 +861,18 @@ test_that("fuzz testing validate_fn()", {
   # Testing side effects
   # Assigning side effects
   side_effects_16417 <- xpectr::capture_side_effects(validate_fn(train_data = dat_ready, formulas = "diagnosis~score+(1|session)", type = "binomial", model_fn = glm_model_fn, predict_fn = glm_predict_fn, test_data = NULL, preprocess_fn = glm_preprocess_fn, preprocess_once = FALSE, hyperparameters = hparams, partitions_col = ".partitions", cutoff = 0.5, positive = 2, metrics = list(all = FALSE, Accuracy = TRUE, Sensitivity = TRUE), rm_nc = FALSE, parallel = FALSE, verbose = FALSE), reset_seed = TRUE)
+
+  # for xpectr::strip(side_effects_16417[['warnings']]), I see
```

Comment:
The optimizers can behave slightly different on different platforms. It's quite annoying :-)
If you enable these tests, I will make sure they are right.